test/TokenBuyer.t.sol

```diff
@@ -201,8 +199,28 @@ contract TokenBuyerTest is Test, IBuyETHCallback {
         assertEq(bot.balance, 1 ether);
     }
 
+    function test_buyETH_paysBackDebt() public {
+        // user has debt of 2000 tokens
+        vm.prank(owner);
+        payer.sendOrRegisterDebt(user, 2000e18);
+        assertEq(payer.debtOf(user), 2000e18);
+
+        // bot buys ETH for 2000 tokens
+        priceFeed.setPrice(2000 * 1e18);
```

Comment:
sometimes we have `2000 * 1e18` and sometimes we have `2000e18`. do you want to keep these different styles or stick with one of them?

Reply:
normalized 17442dfbc75adc7d26978b68618ba0b508f8cf01